<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addEditDialog</class>
 <widget class="QDialog" name="addEditDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>269</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>274</width>
    <height>269</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>23</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QLabel" name="payTypeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>53</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>PayType:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="payTypeComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="noteLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Note:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="noteTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>181</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>addEditDialog</receiver>
   <slot>saveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveData()</slot>
 </slots>
</ui>
